### ISSUE TITLE
Redundant RUSTFLAGS in release.yml?

### ISSUE THREAD

**Issue by suryyyansh** [LlamaEdge/rag-api-server#23]
Are the RUSTFLAGS in [release.yml](https://github.com/LlamaEdge/rag-api-server/blob/main/.github/workflows/release.yml#L32) still required after the addition of the `.cargo` folder?

**Comment by suryyyansh** [comment]
@apepkuss Could I get an update for this. Thanks.

**Comment by apepkuss** [comment]
Sorry for the late reply! Please do not update the release workflow, in case there is any bad effect on the releases. For the potential issue you mentioned above, we'll check it and update the workflow if necessary. Thanks a lot!

**Comment by suryyyansh** [comment]
I understand, but wouldn't it make sense to have an additional release asset for the search enabled `rag-api-server`? since it requires recompilation with the `search` feature to be used.

This is what I was going to include in my [PR](https://github.com/LlamaEdge/rag-api-server/pull/22). Let me know what you think.

**Comment by juntao** [comment]
Yes. I agree that we need release assets for the search-api-server.
